src/mesh.cpp

```diff
@@ -1223,6 +1223,9 @@ StructuredMesh::MeshIndex SphericalMesh::get_indices(
       mapped_r[2] += 2 * M_PI;
   }
 
+  // test
+  coucou = grid_[0]
```

Comment:
I knew a "coucou" or two would turn up one day with you Remi :rofl:

Reply:
![image](https://user-images.githubusercontent.com/40028739/194517709-4f2ce443-6f84-47a6-b301-204af23e5f8e.png)